Phylum/Phylum.manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "author": "Phylum",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "",
+    "helpURL": "https://docs.phylum.io",
```

Comment:
Will there be a doc page specific for this integration ? If not, then we can point to the readme in master branch (similar to how other apps have done).

Reply:
@peterjmorgan - Is the README.md file the source of truth for documentation on this integration? Or are you going to host something on your site? Let's either point to a doc page on your site or this README.md file. Can you update the URL accordingly?

Reply:
Fixed in https://github.com/SumoLogic/sumologic-public-partner-apps/pull/67/commits/ba1271b488f8dd9ee356686c0d4cb260183fe3f5